config/attack-navigator/plaso_tagging_windows_coverage_mitre.json

```diff
@@ -0,0 +1,2445 @@
+{
```

Comment:
IMHO it would be better if we could generate this file from tag_windows.txt to make sure it is kept up to date.

Reply:
Ok you're right, you can delete if you want, the important things is in tag_windows.txt.